_includes/transaction-response.md

```diff
@@ -1,4 +1,4 @@
-{% assign instrument = include.payment_instrument | default: "paymentorder" %}
+{% assign payment_instrument = include.payment_instrument | default: "paymentorder" %}
```

Comment:
You also have to update the rest of this file to use `{{ payment_instrument }}` and not `{{ instrument }}`